app/controllers/volunteers_controller.rb

```diff
@@ -51,10 +51,11 @@ def edit
 
   def update
     authorize @volunteer
-    @volunteer.skip_reconfirmation!
     if @volunteer.update(update_volunteer_params)
-      @volunteer.filter_old_emails!(@volunteer.email)
-      redirect_to edit_volunteer_path(@volunteer), notice: "Volunteer was successfully updated."
+      unless admin_email_change(@volunteer) # this method is in application controller
```

Comment:
I like the idea of the comment but think we can skip the comment.  If this method was defined somewhere _besides_ the `ApplicationController` I would say add a comment but think this is readable enough without it.

Reply:
Removed!